controllers/v1beta1/build_helpers.go

```diff
@@ -731,7 +731,7 @@ func (r *LagoonBuildReconciler) processBuild(ctx context.Context, opLog logr.Log
 			VolumeSource: corev1.VolumeSource{
 				Secret: &corev1.SecretVolumeSource{
 					SecretName:  "lagoon-sshkey",
-					DefaultMode: helpers.IntPtr(420),
+					DefaultMode: helpers.Int32Ptr(420),
```

Comment:
NOTE for self: okay - so practically this doesn't change here, we're just renaming a function to be more precise.

Reply:
Yeah, just a rename because I had to re-implement `IntPtr` to do `int` and it was set for `int32`